Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -105,6 +135,18 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: _ViewController,
                 height: 64
             )
         }
+    
+        (errorOccurredNotificationView as? ChatChannelListErrorView)?.buttonAction = { [weak self] in
+            self?.controller.synchronize()
+        }
+    }
+    
+    func showBottomNotificationIfNeeded(shouldShowNotification: Bool) {
+        errorOccurredNotificationView.isHidden = !shouldShowNotification
+        UIView.animate(withDuration: 0.8) {
+            self.bottomAnimationConstraint?.constant = shouldShowNotification ? -self.errorOccurredNotificationViewHeight : 0
+            self.view.layoutIfNeeded()
+        }
```

Comment:
This function says that `errorOccurredNotificationView` is also a helper for showing bottom notifications... but it's not the case, it's only for showing errors

Also, this is not an "if needed" function, naming needs more thinking

Reply:
Splitted into 2 overridable functions so our customers can customise it better. Also renamed to `show/hide` `BottomErrorNotification`

Reply:
Changed this to a single function with a `Bool` parameter with a more suitable name to match the functionality